### ISSUE TITLE
Opening side panels with key binding throws error

### ISSUE THREAD

**Issue by cyberbit** [Komodo/KomodoEdit#1488]
### Short Summary
Using key bindings to open the left (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>) and right (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>L</kbd>) panes throws `JavaScript Error: "title is not defined"`. Full error is in the info below.

Note that this is only thrown on open using the key binding. Closing them does not throw an error. Toggling from Commando works fine both ways.

### Steps to Reproduce
Press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> or <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>L</kbd>.

### Expected results
Right/left pane toggles with no errors.

### Actual results
An error is thrown on open.

### Platform Information

*Komodo Edit or IDE?* Edit
*Komodo Version?* 10.0.0
*Operating System (and version)?* Windows 7

### Additional Information

```
[Exception... "[JavaScript Error: "title is not defined" {file: "chrome://komodo/content/library/uilayout.js" line: 498}]'[JavaScript Error: "title is not defined" {file: "chrome://komodo/content/library/uilayout.js" line: 498}]' when calling method: [nsIController::doCommand]"  nsresult: "0x80570021 (NS_ERROR_XPC_JAVASCRIPT_ERROR_WITH_DETAILS)"  location: "JS frame :: chrome://komodo/content/library/commands.js :: command_doCommand :: line 272"  data: yes]
                       Tue May 17 2016 10:39:49 GMT-0500 (Central Standard Time)
```

**Comment by Naatan** [comment]
I cannot reproduce this. Do you have addons installed?

**Comment by cyberbit** [comment]
I do. I'll disable and update.

**Comment by cgchoffman** [comment]
Working for me on Windows Edit/IDE.

**Comment by cyberbit** [comment]
I can reproduce this in Safe Mode.

1. Restart in Safe Mode. Use defaults in First Start Wizard.
2. When Komodo loads, wait for Code Intelligence to pre-load.
3. Press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>.

**Comment by Naatan** [comment]
Weird, and you're sure it restarted in safe-mode? (your UI was completely reset)

**Comment by cyberbit** [comment]
Absolutely sure. The Hello Kitty background was gone.

**Comment by Naatan** [comment]
Lol! Alright. Weird!

Does toggling the side panes using the toolbar work?

**Comment by cyberbit** [comment]
Yup, that works fine.

**Comment by mitchell-as** [comment]
Odd, I cannot reproduce this either with the safe-mode steps given...

**Comment by cyberbit** [comment]
I did the same steps on my Windows 10 machine last night. Same results.

**Comment by Naatan** [comment]
@cyberbit since no one but you can reproduce I have to wonder if it could be a corrupt install. Could I ask you to try completely uninstalling Komodo and reinstalling it? Sorry for the inconvenience :\

**Comment by cyberbit** [comment]
@Naatan I'll do it, for science. Would you recommend uninstalling all versions of Komodo as well as 10? Could it be related to a corrupt profile migration?

**Comment by Naatan** [comment]
It should not matter but it certainly wouldn't hurt. It can be a corrupt profile migration but seeing as you said you were able to reproduce this with a clean profile that seems very unlikely.

**Comment by mitchell-as** [comment]
#1546 has additional information on what may be causing this @Naatan @cgchoffman.

**Comment by Naatan** [comment]
Finally been able to reproduce this: It happens on edit only. Also it doesn't seem to actually stop the command from working.

**Comment by cyberbit** [comment]
@Naatan See? I'm not crazy, haha! Probably should have mentioned that it was non-blocking, heh.

**Comment by Naatan** [comment]
I also really gotta pay more attention to the product you're using :)

**Comment by cyberbit** [comment]
Confirmed fixed for me in 10.0.1!

**Comment by tonyattwood** [comment]
Hey,

Have you seen my last game? It's something unbelievable, you've  got to see that! Here  is the link <http://fair.sokrati.in/e4wclqht>

Take care, tony.attwood@ntlworld.com
